src/Sarif/OptionallyEmittedData.cs

```diff
@@ -80,6 +80,9 @@ public enum OptionallyEmittedData : int
         // Enrich SARIF log with git blame information
         GitBlameInformation = 0x1000,
 
+        // Enrich with artifacts only.
+        Artifacts = 0x2000,
```

Comment:
>Artifacts [](http://example.com/codeflow?start=8&length=9)

To keep the same behavior as before, I introduced a new flag that will be able to emit the artifacts only